CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
   [@ikesyo](https://github.com/ikesyo), [#185](https://github.com/AliSoftware/OHHTTPStubs/pull/185)
 * Fix: Carthage support and Examples configurations
   [@Liquidsoul](https://github.com/Liquidsoul), [#190](https://github.com/AliSoftware/OHHTTPStubs/issues/190)
+* Added Swift 3.0 support.
```

Comment:
Tip: please add two spaces after the full-stop, so that the markdown renders a line break and renders the next line as a separate line (otherwise it will join the lines) :wink: